samples/CosmosDbIntegtation/ReadThroughSamples/Models/RedisData.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples.Models
+{
+    public record RedisData(
+        string id,
+        string key,
+        string value,
```

Comment:
I love how you split this up in a `Models` folder, because this is the same for every sample, do you think we could have a folder like `samples/CosmosDbIntergration/Common` or even `samples/CosmosDbIntergration/Models` so we have less shared code?

Reply:
Thank you for moving this, it looks a lot cleaner :)